conf/emonhub.conf

```diff
@@ -43,15 +43,16 @@ loglevel = DEBUG #(default:WARNING)
 [[RFM2Pi]]
     Type = EmonHubJeeInterfacer
     [[[init_settings]]]
-        com_port = /dev/ttyO2
+        #com_port = /dev/ttyAMA0
+	    com_port = /dev/ttyO2
         com_baud = 38400
     [[[runtimesettings]]]
         pubchannels = ToEmonCMS,
         subchannels = ToRFM12,
         
         # datacode = B #(default:h)
         # scale = 100 #(default:1)
-        group = 210 #(default:210)
+        group = 3421749817 #(default:210)
```

Comment:
Why is this a crazy number?

Reply:
This number was kind of encryption, node only send and receive data to another node with this group number.